WalletWasabi.Gui/Tabs/WalletManager/RecoverWalletView.xaml

```diff
@@ -29,6 +29,12 @@
               </DataTemplate>
             </ItemsControl.ItemTemplate>
           </ItemsControl>
+          <CheckBox IsChecked="{Binding ShowAdvancedOptions,Mode=TwoWay}"><TextBlock Text="Show advanced options"/></CheckBox>
+          <StackPanel Spacing="8" IsVisible="{Binding ShowAdvancedOptions}">
+            <controls:ExtendedTextBox Text="{Binding AccountKeyPath}" Watermark="Account Key Path" UseFloatingWatermark="True"/>
+            <controls:ExtendedTextBox Text="{Binding MinGapLimit}" Watermark="Number of keys to derive" UseFloatingWatermark="True"/>
+            <TextBlock Text="Warning: Deriving more than 50 keys can take a while since their UTXOs will also be synchronized."/>
```

Comment:
Warning is not necessary. In principle warnings are not really needed in "Advanced" mode. But also a normal Wasabi wallet uses thousands of keys, because coinjoins are not very blockspace efficient yet.

```suggestion            
```